project-1/main.py

```diff
@@ -0,0 +1,57 @@
+import gspread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+
+driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
```

Comment:
It's good practice to add comments to your code. It will help future you and your team (if you're in one) to understand your code better. A wise man once said, "When you think you have enough comments, that's when you should add more!"